api/config/routes.rb

```diff
@@ -1,56 +1,40 @@
 Spree::Core::Engine.add_routes do
   namespace :admin do
     resources :users do
-      member do
-        put :generate_api_key
-        put :clear_api_key
-      end
+      put :generate_api_key, :clear_api_key, on: :member
```

Comment:
The single `put` method should speed up ruby by 1 nanosecond since there's less methods being called, so I could argue performance, but that wouldn't help.

I like how the blocks, being removed, free up lines of code, but I also like how you have organized your `put` method. I'll opt for that and get best of both worlds.